### PR TITLE
Wielding update and RPG change

### PR DESCRIPTION
**Pull request by TheShown911**
## About The Pull Request
Updated PR with less commits, adds wielding requirements to more guns, and updates the RPG sprite. All feedback is wellcome.

## Details

<details>
  <summary>Expand</summary>

The goal is to update most non handgun weapons to have the wielding requirement.
Guns updated:
-Boltgun (both versions)
-All plasma weapons
-Lightfall
-Lightfall-P
-Cog
-Prototype: laser cannon
-Halicon
-Valkyrie
-X-ray laser
-Granade launchers (IH and NT)
-ALL non pistol (or SMG/Autopistol) projectile weapons.
-Crossbow (both versions)
-AMR Hristov
-ALL shotguns except the sawn off

</details>

## Screenshots

<details>
  <summary>Expand</summary>

Most new RPG sprites:

prP

</details>

### CODE REVIEW

**Comment by TheShown911** [comment]
Will update shotguns per request from the community.

File: code/modules/projectiles/gun.dm [hunk 1]
```diff
@@ -187,7 +188,9 @@
 			return FALSE
 	if(twohanded)
 		if(!wielded)
-			to_chat(user, SPAN_DANGER("The gun is too heavy to shoot in one hand"))
+			if (world.time >= recentwield + 10)
```
Comment (MadmanMartian) [inline code/modules/projectiles/gun.dm]: Change 10 to 1 SECONDS

Easier to read

Comment (TheShown911) [reply code/modules/projectiles/gun.dm]: Done

**Comment by Jan-Blasiak** [comment]
B-e-a-u-t-i-f-u-l

**Pull request merged**